packages/date-time-picker/test/validation.test.js

```diff
@@ -60,16 +60,395 @@ const fixtures = {
       expect(dateTimePicker.invalid).to.equal(false);
     });
 
-    it('should validate on date-picker blur', () => {
+    ['date-picker', 'time-picker'].forEach((pickerType) => {
```

Comment:
Let's create a `value-commit.test.js` file and move all new tests there. Validation and change events are both part of the value commit flow, so testing them together in one place would make sense. Example:

https://github.com/vaadin/web-components/blob/7c2b6e54be7471f6e34b6c7495f3dfd111fe0d3e/packages/date-picker/test/value-commit.test.js#L12-L43

Reply:
Done.